test/parallel/test-http-parser-consume.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+const spawn = require('child_process').spawn;
+
+if (process.argv[2] === 'child') {
+  const server = http.createServer(function(req, res) {
+    res.end('hello');
+  });
+
+  server.listen(0, common.mustCall(function(s) {
+    const rr = http.get({ port: server.address().port },
+                        common.mustCall(function(d) {
+      rr.parser.consume(0);
+      server.close();
+    }));
+  }));
+} else {
+  const child = spawn(process.argv[0], [process.argv[1], 'child'],
+                      { stdio: 'inherit' });
+  child.on('exit', common.mustCall(function(code, signal) {
+    assert(signal === 'SIGABRT', 
```

Comment:
maybe use `assert.strictEqual()` here?
also checking that code is null can be a good idea.